timelock-impl/src/test/java/com/palantir/atlasdb/timelock/paxos/LocalBatchPaxosAcceptorTests.java

```diff
@@ -128,16 +125,16 @@ public void weProxyAcceptRequests() {
                 WithSeq.of(CLIENT_2, 2)));
     }
 
-    @Test
-    public void throwsConjureRuntime404WhenCacheKeyIsNotFound() throws InvalidAcceptorCacheKeyException {
-        AcceptorCacheKey cacheKey = AcceptorCacheKey.newCacheKey();
-        when(cache.updatesSinceCacheKey(cacheKey))
-                .thenThrow(new InvalidAcceptorCacheKeyException(cacheKey));
-
-        assertThatServiceExceptionThrownBy(() -> resource.latestSequencesPreparedOrAcceptedCached(cacheKey))
-                .hasType(BatchPaxosAcceptorResource.CACHE_KEY_NOT_FOUND)
-                .hasArgs(SafeArg.of("cacheKey", cacheKey));
-    }
+//    @Test
+//    public void throwsConjureRuntime404WhenCacheKeyIsNotFound() throws InvalidAcceptorCacheKeyException {
+//        AcceptorCacheKey cacheKey = AcceptorCacheKey.newCacheKey();
+//        when(cache.updatesSinceCacheKey(cacheKey))
+//                .thenThrow(new InvalidAcceptorCacheKeyException(cacheKey));
+//
+//        assertThatServiceExceptionThrownBy(() -> resource.latestSequencesPreparedOrAcceptedCached(cacheKey))
+//                .hasType(BatchPaxosAcceptorResource.CACHE_KEY_NOT_FOUND)
+//                .hasArgs(SafeArg.of("cacheKey", cacheKey));
+//    }
```

Comment:
We probably do want a version of this test (except that it throws the `InvalidAcceptorCacheKeyException` rather than `ServiceException`) I think?

Reply:
oop my bad, I thought I had made an equivalent test for the resource, will fix up.

Reply:
I changed one of the tests that exercises the cache key not found logic to use the logical exception instead of the remote exception.

I'm going to make a `BatchPaxosAcceptorResourceTest` class, which tests the conversion.